mlflow/keras.py

```diff
@@ -82,9 +82,11 @@ def save_model(keras_model, path, conda_env=None, mlflow_model=Model(), custom_o
                                 'tensorflow=1.8.0'
                             ]
                         }
-    :param keras_module: Keras module to be used to save / load the model. If not provided, MLflow
-    will attempt to infer the Keras module based on the given model.
     :param mlflow_model: MLflow model config this flavor is being added to.
+    :param custom_objects: Keras custom_objects dictionary containing String:Class
+    mappings for custom layers in your Keras model.
+    :param keras_module: Keras module to be used to save / load the model. If not
```

Comment:
Nice catch! It would useful to list the valid modules (`keras` or `tf.keras`) here.